pkg/sysinfo/sysinfo_linux.go

```diff
@@ -17,6 +17,15 @@ const (
 	SeccompModeFilter = uintptr(2)
 )
 
+var previousWarnings = map[string]struct{}{}
```

Comment:
@AkihiroSuda correct me I'm wrong but this would mean I would need to create a global variable (a sync.Once) for each warning. So each warning would need to have two parts to it (global var and then the call to the correct once.Do) - feels more complicated than what I did here.